src/UnitarySimulator.cpp

```diff
@@ -4,17 +4,17 @@
 
 #include <chrono>
 
-template<class DDPackage>
-void UnitarySimulator<DDPackage>::Construct() {
+template<class Config>
+void UnitarySimulator<Config>::Construct() {
     // carry out actual computation
     auto start = std::chrono::steady_clock::now();
     if (mode == Mode::Sequential) {
-        e = dd::buildFunctionality(CircuitSimulator<DDPackage>::qc.get(), Simulator<DDPackage>::dd);
+        e = dd::buildFunctionality(CircuitSimulator<Config>::qc.get(), Simulator<Config>::dd);
     } else if (mode == Mode::Recursive) {
-        e = dd::buildFunctionalityRecursive(CircuitSimulator<DDPackage>::qc.get(), Simulator<DDPackage>::dd);
+        e = dd::buildFunctionalityRecursive(CircuitSimulator<Config>::qc.get(), Simulator<Config>::dd);
     }
     auto end         = std::chrono::steady_clock::now();
     constructionTime = std::chrono::duration<double>(end - start).count();
 }
 
-template class UnitarySimulator<dd::Package<>>;
+template class UnitarySimulator<dd::DDPackageConfig>;
```

Comment:
Nothing for now, but to keep in mind for the future: It would definitely make sense to provide a dedicated DDPackage configuration for this simulator as it does not use vectors at all, but heavily uses matrices. Increasing/decreasing the respective compute and unique table sizes could help here.

Reply:
Just to make sure: you noted that somewhere?
Consider just creating an issue for things like these. Helps to keep them on the radar.

Reply:
See https://github.com/cda-tum/ddsim/issues/182